src/scss/03_organisms-blocks/_widget.scss

```diff
@@ -1,22 +1,40 @@
+// stylelint-disable media-feature-name-whitelist
+
 .widget {
+  $root: &;
+
   h2,
   h3 {
     color: $color-secondary;
     font-size: 17px;
     font-weight: 700;
+
+    @media (max-width: 767px) {
+      color: $color-blue-19;
+      font-family: $font-secondary;
+      font-size: 18px;
+      font-weight: 700;
+    }
   }
 
   &--alt {
-    border: 2px solid darken($color-alt, 12%);
     text-align: center;
 
+    #{$root} {
+      &__border {
```

Comment:
Should this really require `&__border`?
Not sure if it's a difference in my local but I **think** the extra `.widget__border` was basically an extra wrapper that only applied the border styling, so that's why it was in the previous place directly applied to `&--alt`

Reply:
Correct, it is an extra class, but it;s so that the border doesn't wrap around the outer element's padding ( from column styles ).